lib/ex_admin/ex_admin.ex

```diff
@@ -237,7 +210,7 @@ defmodule ExAdmin do
 
   @doc false
   def get_title_actions(%Plug.Conn{private: _private, path_info: path_info} = conn) do
-    defn = get_registered_by_controller_route(path_info)
+    defn = conn.assigns.defn
```

Comment:
On the Spree example, I can say that namespacing can cause more harm than good. Much extra work and only theoretic benefits.